firefox-ios/firefox-ios-tests/Tests/Smoketest1.xctestplan

```diff
@@ -15,13 +15,17 @@
   "testTargets" : [
     {
       "skippedTests" : [
+        "ActivityStreamTest",
         "ActivityStreamTest\/testContextMenuInLandscape()",
+        "ActivityStreamTest\/testDefaultSites()",
```

Comment:
@dragosb01 We are removing tests from the SMoketest 1, are those in other test suites already? I see we have 22 tests in smoketest 1, if we land this PR we will have 15

Reply:
@isabelrios the tests were moved in Smoketest4:
https://addons-testing.bitrise.io/builds/82d437d5-f56d-432b-af9b-34eb52053306/testreport/1b78e3fc-354a-4534-8fbc-1ebda9dcf98e/testsuite/0/testcases?status=failed
The tests are still executed under our smoke plans.

Reply:
As you can see we have 7 more tests on Smoketest4, the difference from 22 to 15 from Smoketest1

Reply:
oh I see yeah, I saw that.. the smoketest 1 takes longer than the others..